Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rocker/r-ver:latest
+
+RUN Rscript -e 'install.packages("remotes")'
+
+COPY DESCRIPTION /bound/DESCRIPTION
```

Comment:
NB: I would choose a different directory name than `bound`. That's not  very descriptive. Perhaps `pacta_analysis`?

Reply:
Unfortunately, there are some hard-coded expectations in `PACTA_analysis`, that make it run a certain way if it detects that it is being run in a folder called `bound`. (This is to "detect that it is in transition monitor" or something like that). The least intrusive way around that is to follow the same protocol that Transition Monitor does, to ensure the tool runs the same in both environments.